src/components/Mission.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { bookMission, unbookMission } from '../redux/missions/mission';
+import '../styles/mission.css';
+
+const Mission = (props) => {
+  const {
+    id, name, description, reserved,
+  } = props;
+
+  const dispatch = useDispatch();
+  const reserve = () => {
+    dispatch(bookMission(id));
+  };
```

Comment:
- [ ] After I reserve a rocket or join a mission, I can find the reserved rockets and joined missions on the profile page. Great work👍. However, all of the reserved rockets and joined missions vanished when I navigated back to the missions or rockets page.
Please ensure that the state doesn't return to its initial condition while navigating between pages.